tests/integration/tier0/non-destructive/rollback-handling/test_rollback_handling.py

```diff
@@ -256,4 +269,24 @@ def test_terminate_registration_success(convert2rhel):
         c2r.expect("DEBUG - Calling command 'subscription-manager attach --auto'", timeout=180)
         c2r.expect("Status:       Subscribed", timeout=180)
         terminate_and_assert_good_rollback(c2r)
-    assert c2r.exitstatus != 0
+    assert c2r.exitstatus == 1
+
+
+@pytest.mark.test_rollback_failure
+def test_rollback_failure(shell, convert2rhel, immutable_os_release_file):
+    """
+    Make os-release file immutable. This cause the conversion rollback to fail (https://issues.redhat.com/browse/RHELC-1248).
+    Verify that the `analyze` ends with exit code 1 respecting the failure (https://issues.redhat.com/browse/RHELC-1275).
```

Comment:
```suggestion
    Make os-release file immutable. This cause the conversion rollback to fail (https://issues.redhat.com/browse/RHELC-1248).
    Verify that the `analyze` ends with exit code 1 respecting the failure (https://issues.redhat.com/browse/RHELC-1275).
```
```suggestion
    Make os-release file immutable. This will cause the conversion rollback to fail (https://issues.redhat.com/browse/RHELC-1248).
    Verify that the analysis ends with exit code 1 respecting the failure (https://issues.redhat.com/browse/RHELC-1275).
```